packages/core/src/dialects/db2/query.js

```diff
@@ -36,152 +36,154 @@ export class Db2Query extends AbstractQuery {
 
     const complete = this._logQuery(sql, debug, parameters);
 
-    const errStack = new Error().stack;
-
-    return new Promise((resolve, reject) => {
-      // TRANSACTION SUPPORT
-      if (_.startsWith(this.sql, 'BEGIN TRANSACTION')) {
-        connection.beginTransaction(err => {
-          if (err) {
-            reject(this.formatError(err, errStack));
-          } else {
-            resolve(this.formatResults());
-          }
-        });
-      } else if (_.startsWith(this.sql, 'COMMIT TRANSACTION')) {
-        connection.commitTransaction(err => {
-          if (err) {
-            reject(this.formatError(err, errStack));
-          } else {
-            resolve(this.formatResults());
-          }
-        });
-      } else if (_.startsWith(this.sql, 'ROLLBACK TRANSACTION')) {
-        connection.rollbackTransaction(err => {
-          if (err) {
-            reject(this.formatError(err, errStack));
-          } else {
-            resolve(this.formatResults());
-          }
-        });
-      } else if (_.startsWith(this.sql, 'SAVE TRANSACTION')) {
-        connection.commitTransaction(err => {
-          if (err) {
-            reject(this.formatError(err, errStack));
-          } else {
-            connection.beginTransaction(err => {
-              if (err) {
-                reject(this.formatError(err, errStack));
-              } else {
-                resolve(this.formatResults());
-              }
-            });
-          }
-        }, this.options.transaction.name);
-      } else {
-        const params = [];
-        if (parameters) {
-          _.forOwn(parameters, (value, key) => {
-            const param = this.getSQLTypeFromJsType(value, key);
-            params.push(param);
-          });
-        }
+    if (this.sql.startsWith('BEGIN TRANSACTION')) {
```

Comment:
Not really related to this PR, but the imports of `util` and `dayjs` are unused in this file.

Reply:
Done

Reply:
Hmm, I thought these were still pending. But apparently not